usagov/usagov-2021
# USAGOV-1207-Icon-List-Alt-Text

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1207

## Description

Add aria-label. Remove aria-hidden.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


* visit /employer-fmla
* inspect the list icons
* verify that the SVG element has a descriptive aria-label attribute and does not have the aria-hidden attribute.
* also check the Spanish page

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
